app-config-exec/src/index.ts

```diff
@@ -0,0 +1,104 @@
+import { Json, parseRawString, ParsingExtension } from '@lcdev/app-config';
+import { guessFileType } from '@lcdev/app-config/dist/config-source';
```

Comment:
I feel like we should export this function from top-level if plugins find it useful